bridge-proxy/tests/test-contract/crowdfunding-esdt.mxsc.json

```diff
@@ -0,0 +1,196 @@
+{
+    "buildInfo": {
+        "rustc": {
+            "version": "1.80.0-nightly",
```

Comment:
why nightly?

Reply:
Fixed. Replaces with stable and latest framework (0.52.3).